Products/CMFPlone/resources/browser/resource.py

```diff
@@ -87,6 +87,11 @@ def update(self):
         # collect names
         js_names = [name for name, rec in records.items() if rec.jscompilation]
         css_names = [name for name, rec in records.items() if rec.csscompilation]
+        # Allow bundles to have "theme" resp. "theme" and "custom" as dependency
+        # so that bundles can be rendered after theme and custom resources.
+        js_names += ["theme"]
+        css_names += ["theme", "custom"]
```

Comment:
See below in https://github.com/plone/Products.CMFPlone/pull/4054/files#diff-9e095767a08db89a248c84c46d31c75898eed2d7dc40b0757a835bb2af0ed95bR143 and https://github.com/plone/Products.CMFPlone/pull/4054/files#diff-9e095767a08db89a248c84c46d31c75898eed2d7dc40b0757a835bb2af0ed95bR165

If "theme" or "custom" does not exist (both added in line 192+) the bundle dependency cannot be resolved and the bundle is not included in the rendered output.